content/blog/pytest-testing-with-nuttx/index.md

```diff
@@ -0,0 +1,369 @@
+---
+title: "Testing applications with Pytest and NuttX"
+date: 2024-10-04T16:54:43-03:00
```

Comment:
You can use the date format as `YYYY-MM-DD` without the timestamp.
Please change to `2024-10-04`; otherwise, the article will be visible only at 4 pm UTC. Hugo does not render articles for which the date and time were set to a future date.

Reply:
Oh I see. Sorry about that.
Done.